Documentation/help_files/history_dashboard_help.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * History Dashboard Help.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @version 1.0.0
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+ 
+use OpenEMR\Core\Header;
+
+require_once("../../interface/globals.php");
+?>
+<!DOCTYPE HTML>
+<html>
+    <head>
+    <?php Header::setupHeader();?>
+    <title><?php echo xlt("History and Lifestyle Help");?></title>
+    </head>
+    <body>
+        <div class="container oe-help-container">
+            <div>
+                <center><h2><a name='entire_doc'><?php echo xlt("History and Lifestyle Help");?></a></h2></center>
+            </div>
+            <div class= "row">
+                <div class="col-sm-12">
+                    <p><?php echo xlt("A place to document and display the patient's past medical history, family history, personal history");?>.
+                    
+                    <p><?php echo xlt("It has four tabs");?>:
+                        <ul>
+                            <li><?php echo xlt("General"); ?></li>
+                            <li><?php echo xlt("Family History"); ?></li>
+                            <li><?php echo xlt("Relatives"); ?></li>
+                            <li><?php echo xlt("Lifestyle"); ?></li>
+                            <li><?php echo xlt("Other"); ?></li>
```

Comment:
Note that the history layout is completely customizable in the Layouts editor, so recommend being much less specific on this howto document.

Reply:
How to applies for default. Can we add some wording about customization. Is there a wiki entry I could point to ?